com.br_automation.buoat.xddeditor.editor/src/com/br_automation/buoat/xddeditor/editor/adapters/SubObjectPropertySource.java

```diff
@@ -95,6 +95,8 @@
     public static final String INVALID_WO_TPDO_MAPPING = "Sub-object with access type 'wo' does not allow TPDO mapping";
     public static final String INVALID_RO_RPDO_MAPPING = "Sub-object with access type 'ro' does not allow RPDO mapping";
     public static final String NO_CHANGE_IN_DATA_TYPE = "No change in data type.";
+    public static final String WARNING_ARRAY = "Changing the data type will remove the current values in 'Default value', 'Low limit' and 'High Limit' of all sub-Objects in the array.\n\nAre you sure you want to change?";
```

Comment:
sub-objects

Reply:
We will update the changes in the next set of bug fix.